tests/test_form.py

```diff
@@ -106,7 +106,7 @@ def test_choose_submit(expected_post):
     pytest.param('continue', id='first'),
     pytest.param('cancel', id='second'),
 ])
-def test_choose_submit_from_selector(value):
+def test_choose_submit_from_selector_submit(value):
```

Comment:
Why this renaming?

Reply:
Because I basically replicated this same test, but for "image" type.

Reply:
Ah, OK.